jcl/src/com.ibm.management/share/classes/com/ibm/lang/management/RuntimeMXBean.java

```diff
@@ -75,6 +75,20 @@ public String idleStateName() {
 	 */
 	public double getCPULoad();
 
+	/*[IF Java10]*/
+	/**
+	 * Returns the native process identifier that identifies the current
+	 * Java process to the operating system. The value is prone to being 
+	 * recycled over a period of time, as considered suitable by the 
+	 * operating system.
+	 * 
+	 * @return A long representing the process ID (pid) on the underlying 
+	 * operating system.
+	 * 
+	 * @deprecated Use getPid() instead.
+	 */
+	@Deprecated
```

Comment:
Should be @Deprecated(forRemoval = true, since = "10")

Reply:
Also, please don't duplicate the entire javadoc just to add the comment about deprecation, you can use preprocessor tags to modify the single copy of the javadoc.